### ISSUE TITLE
Feature request (please): smooth lines transition for auto generated subtitles

### ISSUE THREAD

**Issue by ChuckBartowski91** [yuliskov/SmartTube#2203]
I would like that the subtitles automatically generated could flow like the normal subtitles, because now every line pops out flashing, which can cause strain to the eyes (in the youtube app they show word by word, not very good but does't cause eyestrain).

**Comment by JMskch** [comment]
Yes, please do something with autogenerated subtitles. Very bad to read.
